lib/widgets/content.dart

```diff
@@ -730,12 +730,13 @@ class MessageImageEmoji extends StatelessWidget {
           // Web's css makes this seem like it should be -0.5, but that looks
           // too low.
           top: -1.5,
-          child: RealmContentNetworkImage(
-            resolvedSrc,
-            filterQuality: FilterQuality.medium,
-            width: size,
-            height: size,
-          )),
+          child: resolvedSrc == null ? const SizedBox.shrink() // TODO(log)
+            : RealmContentNetworkImage(
+                resolvedSrc,
+                filterQuality: FilterQuality.medium,
+                width: size,
+                height: size,
+              )),
```

Comment:
For emoji reactions, if the src doesn't parse, we fall back to showing the emoji name. (We use the same fallback if the src parses but the image fails to load.) This would be nice to do, or we could add a TODO for it.

I wonder if it would be hard to factor the emoji-reaction widget code into something to also use in content. Another thing I notice is that `_UnicodeEmoji` in widgets/emoji_reaction.dart has careful logic so that the emoji isn't awkwardly small on iOS. (In content, unicode emojis are awkwardly small on iOS.)

Reply:
Yeah, it'd be good to reuse the work that went into those reaction emoji so that emoji in content also get the benefit of it. It looks like features that would bring include:
* making Unicode emoji bigger on iOS, counteracting that Flutter-and/or-iOS bug;
* disabling animations based on the platform setting;
* text fallbacks based on the user's emojiset choice;
* text fallbacks when an image doesn't work out.

At the moment we don't have an implementation of emoji-as-text-name at all for content, so I think doing that as a fallback is out of scope for this PR. The URL failing to parse should happen rarely to never — it'd be a bug in the server, and doesn't seem like a real likely kind of bug.

Reply:
Makes sense.